CONTRIBUTING.md

```diff
@@ -123,11 +129,21 @@ See the next section for how to adapt to upstream changes.
 
 ### Branching Strategy
 
-By default, development happens in branches, which are merged via pull requests.
+By default, development happens in branches, which are merged via pull requests (PRs from `master` will be closed).
 Special cases, like fixing problems with the CI pipeline, are of course exempt from this guideline.
 
 Please make sure to give branches a meaningful name!
 As an example, the one creating this documentation was called `branching-merging-documentation`.
+For bonus points, adhere to the following rule.
+
+For branches that are pushed directly to this repo (take note maintainers!), this naming strategy is obligatory:
+
+* branches intended to fix an issue must be named `issue/$NUMBER-$TITLE` where `$NUMBER` is the issue number on GitHub and `$TITLE` a very short summary of what the issue is about (for those of us who don't know all issues by heart) in lower case letters and dash-separated, e.g. `issue/226-team-of-maintainers`
```

Comment:
1. New sentences start with a upper letter, even after `:` ;)
2. I really appreciate the naming, but why are we more strict with names of (temporal existing) branches over names of (permanently existing) classes?

> Note should, not must - there can be exceptions if well argued

edit: 5 mins later : Hmm this only applies for branches that are pushed directly to this repo - which can only be done by maintainers, right? Then we should move the "take note maintainers" as a header to make clear that this naming thing is a `MUST` for maintainers but an `YOU SHOULD` for "free" contributors.

edit2: another 15mins later:
And that's somehow contradictory as we write at the beginning:

> The guidelines apply to maintainers as well as contributors!

Reply:
I don't like simple lists, whose items often aren't even real sentences to start with uppercase letters. So they don't. :grin:

Reply:
The naming rule applies to all branches that are pushed to this repo - by "coincidence" only maintainers can do that, but the rule still applies to everyone in the sense that few people can do the prerequisite action (push to the repo). I put the rule here to encourage all contributors to adhere to it, which would make our live easier.